src/config.c

```diff
@@ -274,3 +310,95 @@ int load_config(struct vpn_config *cfg, const char *filename)
 
 	return ret;
 }
+
+void destroy_vpn_config(struct vpn_config *cfg)
+{
+	free(cfg->pppd_log);
+	free(cfg->pppd_plugin);
+	free(cfg->pppd_ipparam);
+	free(cfg->pppd_ifname);
+	free(cfg->pppd_call);
```

Comment:
good catch, `pppd_call` was missing in the [original code](https://github.com/adrienverge/openfortivpn/blob/master/src/main.c#L129)